ros2_serial_example/src/ros2_to_serial_bridge_main.cpp

```diff
@@ -38,7 +38,9 @@ int main(int argc, char *argv[])
     std::shared_ptr<ros2_to_serial_bridge::ROS2ToSerialBridge> node;
     try
     {
-        node = std::make_shared<ros2_to_serial_bridge::ROS2ToSerialBridge>();
+        rclcpp::NodeOptions node_options;
+        node_options.allow_undeclared_parameters(true).automatically_declare_parameters_from_overrides(true);
```

Comment:
Instead of setting `allow_undeclared_parameters` to true, I think it would be much better to switch from `get_parameter` to `declare_parameter` below.  That's much more idiomatic on Dashing.